unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/ActionController/ActionController.cs

```diff
@@ -88,6 +88,13 @@ public void TryToUndoAction()
 
     }
 
+    public void CreateActionEntityDeleted(DCLBuilderInWorldEntity entity)
+    {
+        List<DCLBuilderInWorldEntity> entityList = new List<DCLBuilderInWorldEntity>();
+        entityList.Add(entity);
+        CreateActionEntityDeleted(entityList);
+    }
```

Comment:
Do we need to create a new list here?

Reply:
This is a special case when only 1 entity is deleted, in order to maintain the same structure to enforce that you always apply changes to the list of selected entities a new list with 1 element is made, but maybe it could be done in a way that doesn't require a major refactoring of the class for 1 special case. If you see a way of doing it, let me know!